docs/05-infrastructure-nodes.md

```diff
@@ -0,0 +1,326 @@
+# OpenShift Infrastructure Nodes
+The OpenShift subscription model allows customers to run various core
```

Comment:
Hmm, do you think installer repo is the right place to point this BZs behavior?

Reply:
We can define `Infrastructure Nodes` as node that run OpenShift core components, which include
list at https://github.com/openshift/training/pull/426/files#diff-37208a9588b8cdc5a7b273b09418a3c3R11

and keep the subscription out?

Reply:
I can't agree with you here. The whole point of presenting the commentary on infrastructure nodes is because this is a common scenario that OCP4 customers are going to need to work through. The purpose of the training repo is to guide customers through common scenarios with OpenShift 4. Without the background on subscriptions and _why_ they would create infrastructure nodes I think the exercise would seem like a waste of time. But it's important.